CondFormats/EcalObjects/interface/EcalTimeBiasCorrectionsGPU.h

```diff
@@ -0,0 +1,49 @@
+#ifndef CondFormats_EcalObjects_interface_EcalTimeBiasCorrectionsGPU_h
+#define CondFormats_EcalObjects_interface_EcalTimeBiasCorrectionsGPU_h
+
+#include "CondFormats/EcalObjects/interface/EcalTimeBiasCorrections.h"
+#include "FWCore/Utilities/interface/propagate_const_array.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/device_unique_ptr.h"
+
+#ifndef __CUDACC__
+#include "HeterogeneousCore/CUDAUtilities/interface/HostAllocator.h"
+#include "HeterogeneousCore/CUDACore/interface/ESProduct.h"
+#endif  // __CUDACC__
+
+class EcalTimeBiasCorrectionsGPU {
+public:
+  struct Product {
+    edm::propagate_const_array<cms::cuda::device::unique_ptr<float[]>> EBTimeCorrAmplitudeBins;
+    edm::propagate_const_array<cms::cuda::device::unique_ptr<float[]>> EBTimeCorrShiftBins;
+    edm::propagate_const_array<cms::cuda::device::unique_ptr<float[]>> EETimeCorrAmplitudeBins;
+    edm::propagate_const_array<cms::cuda::device::unique_ptr<float[]>> EETimeCorrShiftBins;
+    int EBTimeCorrAmplitudeBinsSize, EETimeCorrAmplitudeBinsSize;
```

Comment:
EB, EE -> eb, ee to stick with https://cms-sw.github.io/cms_coding_rules.html 2.9 (sorry)